acj/api/classlist.py

```diff
@@ -219,6 +221,10 @@ def import_users(import_type, course, users):
         db.session.add(enrolment)
     db.session.commit()
 
+    # wait until user ids are generated before starting background jobs
+    for (user, password) in set_user_passwords:
```

Comment:
Is it creating a 1000 tasks if I have 1000 users with passwords? Maybe it is better to move the loop inside the task? We probably will never need to run a task just updating  a single password. Even if we do, we can put it in an array.

Is the parameter for a task has to be serialisable?